packages/ast-spec/src/declaration/ClassDeclaration/fixtures/implementsMany/snapshots/8-AST-Alignment-Tokens.shot

```diff
@@ -0,0 +1,112 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AST Fixtures declaration ClassDeclaration implementsMany AST Alignment - Token 1`] = `
+"Snapshot Diff:
+- TSESTree
++ Babel
+
+  Array [
+    Keyword {
+      type: 'Keyword',
+      value: 'class',
+
+      range: [0, 5],
+      loc: {
+        start: { column: 0, line: 1 },
+        end: { column: 5, line: 1 },
+      },
+    },
+    Identifier {
+      type: 'Identifier',
+      value: 'Foo',
+
+      range: [6, 9],
+      loc: {
+        start: { column: 6, line: 1 },
+        end: { column: 9, line: 1 },
+      },
+    },
+-   Keyword {
+-     type: 'Keyword',
++   Identifier {
++     type: 'Identifier',
+      value: 'implements',
+
+      range: [10, 20],
+      loc: {
+        start: { column: 10, line: 1 },
+        end: { column: 20, line: 1 },
+      },
+    },
+    Identifier {
+      type: 'Identifier',
+      value: 'Object',
+
+      range: [21, 27],
+      loc: {
+        start: { column: 21, line: 1 },
+        end: { column: 27, line: 1 },
+      },
+    },
+    Punctuator {
+      type: 'Punctuator',
+      value: ',',
+
+      range: [27, 28],
+      loc: {
+        start: { column: 27, line: 1 },
+        end: { column: 28, line: 1 },
+      },
+    },
+    Identifier {
+      type: 'Identifier',
+      value: 'Function',
+
+      range: [29, 37],
+      loc: {
+        start: { column: 29, line: 1 },
+        end: { column: 37, line: 1 },
+      },
+    },
+    Punctuator {
+      type: 'Punctuator',
+      value: ',',
+
+      range: [37, 38],
+      loc: {
+        start: { column: 37, line: 1 },
+        end: { column: 38, line: 1 },
+      },
+    },
+    Identifier {
+      type: 'Identifier',
+      value: 'RegExp',
+
+      range: [39, 45],
+      loc: {
+        start: { column: 39, line: 1 },
+        end: { column: 45, line: 1 },
+      },
+    },
+    Punctuator {
+      type: 'Punctuator',
+      value: '{',
+
+      range: [46, 47],
+      loc: {
+        start: { column: 46, line: 1 },
+        end: { column: 47, line: 1 },
+      },
+    },
+    Punctuator {
+      type: 'Punctuator',
+      value: '}',
+
+      range: [47, 48],
+      loc: {
+        start: { column: 47, line: 1 },
+        end: { column: 48, line: 1 },
+      },
+    },
+  ]"
+`;
```

Comment:
example of an token misalignment.

[**Rendered in GH**](https://github.com/typescript-eslint/typescript-eslint/blob/63f5b1ab8a27cb0daf12a325988ffe674c3267be/packages/ast-spec/src/declaration/ClassDeclaration/fixtures/implementsMany/snapshots/8-AST-Alignment-Tokens.shot)